modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/resources/app.conf

```diff
@@ -49,4 +49,5 @@ seqexec-engine {
     gcalKeywords = false
     instForceError = false
     odbQueuePollingInterval = "10 seconds"
+    tops = "tcs=tcs:, ao=ao:, gm=gm:, gc=gc:, gws= ws:, m2=m2:, oiwfs=oiwfs:, ag=ag:, f2=f2:"
```

Comment:
I decided against making a parameter for each one because I'm lazy. This format as the drawback that one typo may break all the tops.